src/partials/language/en.html

```diff
@@ -27,8 +27,10 @@
   "clipboard.copy": "Copy to clipboard",
   "clipboard.copied": "Copied to clipboard",
   "edit.link.title": "Edit this page",
+  "footer.title": "Footer",
```

Comment:
Minor issue: please ensure that keys are sorted alphabetically.

Reply:
Alright, I'll fix the rest of the keys also as they were not in order.

Reply:
Ah wait. Okay, alphabetical order makes no sense, as `language` and `directionality` should be at the top. Nevermind, then.